### PR TITLE
Run unit tests on several browsers on Sauce Labs

### PR DESCRIPTION
**Pull request by xaviervia**
Just opening @cpapazaf 's branch for keeping track of the fixes for each of the failing browsers. Please review the setup, if there are no comments I'll merge once the builds are green.

### CODE REVIEW

File: tests/propTypes/validateSize.spec.es6 [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-import validateSize from 'propTypes/validateSize'
+import validateSize from '../propTypes/validateSize'
 import assert from 'assert'
 
 const invalidSizes = [1, {}, [], '1.', '', 'px', '1p', '%', 'wrong', 'em2']
 const validSizes = [undefined, '0', '5px', '100%', '3.5em', '12rem', '0ex', '7.55cm', '300mm', '2in', '10pt', '3pc', '11ch', '77vh', '0.4vw', '.1vmin', '.2vmax']
 
-describe('validateSize', () => {
+describe.only('validateSize', () => {
```
Comment (joaomilho) [inline tests/propTypes/validateSize.spec.es6]: don't forget to remove the only

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
 node_js:
-- '4'
 - '5'
 before_script:
 - npm install
 script:
-- npm run lint
-- npm test
+  - npm run lint
+  - BROWSER=SL_IE_10,SL_IE_11,SL_IE_13,SL_MAC_SAFARI_9_0,SL_Android_4_3,SL_Android_5_0,SL_IOS_9_2,PhantomJS npm test
```
Comment (joaomilho) [inline .travis.yml]: cool

File: tests/propTypes/validateSize.spec.es6 (hunk 1 above)
Comment (xaviervia) [inline tests/propTypes/validateSize.spec.es6]: Sure. This is a WIP PR. I will mark it as such. =)

**Comment by xaviervia** [comment]
screen shot 2016-05-10 at 17 44 37

This one is always failing, I'm skipping it.

**Pull request merged**